inference-engine/src/vpu/common/include/vpu/configuration/options/performance_hint.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
These files (options/..) should be moved since the PR#9110 has been merged. I believe the new directory for these files should be `openvino/src/plugins/intel_vpu/common/include/vpu/configuration/options`.

Reply:
Thanks. I did that, but now I'm waiting for a local build to test it.